securedrop-export/debian/copyright

```diff
@@ -2,26 +2,5 @@ Upstream-Name: securedrop-export
 Source: https://github.com/freedomofpress/securedrop-export
 
 Files: *
-Copyright: 2019 Freedom of the Press Foundation
-License: GPL-3.0+
-
-Files: debian/*
-Copyright: 2019 Freedom of the Press Foundation <securedrop@freedom.press>
-License: GPL-3.0+
-
-License: GPL-3.0+
- This program is free software: you can redistribute it and/or modify
- it under the terms of the GNU General Public License as published by
- the Free Software Foundation, either version 3 of the License, or
- (at your option) any later version.
- .
- This package is distributed in the hope that it will be useful,
- but WITHOUT ANY WARRANTY; without even the implied warranty of
- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- GNU General Public License for more details.
- .
- You should have received a copy of the GNU General Public License
- along with this program. If not, see <https://www.gnu.org/licenses/>.
- .
- On Debian systems, the complete text of the GNU General
- Public License version 3 can be found in "/usr/share/common-licenses/GPL-3".
+Copyright: 2020 Freedom of the Press Foundation
+License: AGPL-3.0+
```

Comment:
woops noticed this was not AGPL 3: https://github.com/freedomofpress/securedrop-export/pull/45